### PR TITLE
Map fixes

### PR DESCRIPTION
**Pull request by evulins**
- when "Satellite" is selected, "Labels" should be on by default
- desktop: map to satellite toggle: the alignment of the white dot within the blue background is off, see attached screenshot
- mobile: map selector text should be black and up/down arrows should be replaced by down arrow
- mobile: hamburger icon should be vertically centered within white background, see attached screenshot for reference

Trello cards:
[Display map](https://trello.com/c/waQ98X5U/1813-display-map)
[Hamburger menu](https://trello.com/c/3UtFPzRO/1822-habitatmap-hamburger)

### CODE REVIEW

File: app/javascript/react/components/Map/mapStyles.ts [hunk 1]
```diff
@@ -0,0 +1,222 @@
+export default [
+    {
+      featureType: "administrative",
+      elementType: "labels.text.fill",
+      stylers: [
+        {
+          color: "#acacac",
```
Comment (smialko) [inline app/javascript/react/components/Map/mapStyles.ts:7]: plaese, use Colors

File: app/javascript/react/components/Map/mapStyles.ts [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        {
+          visibility: "off",
+        },
+      ],
+    },
+    {
+      featureType: "landscape",
+      stylers: [
+        {
+          color: "#f2f2f2",
```
Comment (smialko) [inline app/javascript/react/components/Map/mapStyles.ts:40]: same as above :D

File: app/javascript/react/components/Map/mapStyles.ts [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+          visibility: "simplified",
+        },
+      ],
+    },
+    {
+      featureType: "landscape.man_made",
+      elementType: "geometry.stroke",
+      stylers: [
+        {
+          color: "#c9c9c9",
```
Comment (smialko) [inline app/javascript/react/components/Map/mapStyles.ts:75]: :D

File: app/javascript/react/components/Map/mapStyles.ts [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+          visibility: "simplified",
+        },
+      ],
+    },
+    {
+      featureType: "road.highway",
+      elementType: "geometry.fill",
+      stylers: [
+        {
+          color: "#ffffff",
```
Comment (smialko) [inline app/javascript/react/components/Map/mapStyles.ts:172]: colors please

File: app/javascript/react/components/Map/mapStyles.ts [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+        {
+          visibility: "off",
+        },
+      ],
+    },
+    {
+      featureType: "water",
+      stylers: [
+        {
+          color: "#e0f1f9",
```
Comment (smialko) [inline app/javascript/react/components/Map/mapStyles.ts:206]: Colors please

**Review by smialko — APPROVED** [review]
Just minor things :)

**Review by JoannaThelema — APPROVED** [review]
🙌

**Pull request merged**